cocos/ui/deprecated.ts

```diff
@@ -56,105 +56,120 @@ legacyCC.UIReorderComponent = UIReorderComponent;
 
 /**
  * Alias of [[Button]]
- * @deprecated Since v1.2
+ * @deprecated Since v3.6.0, please use `Button` instead.
```

Comment:
actually we deprecated these API since v1.2, 
but for developers, all they know is that they should not use them since v3.6.0